packages/database/src/migrations/20210623222559-AddNumSecondaryClassroomsLESMIS-modifies-data.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, deleteObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_CODE = 'LESMIS_ESSDP_LowerSecondarySubSector_Schools';
+const CODE = 'LESMIS_num_secondary_classrooms';
+
+const REPORT_CONFIG = {
+  fetch: {
+    dataElements: ['noclassroom_public_se', 'noclassroom_private_se'],
```

Comment:
Do these need to be swapped from `_se` to `_lse` ? There's a discussion about it on the ticket and I'm a little confused what the outcome was

Reply:
Absolutely they do. Good pickup, swapped